backend/apis/clients/client-logs/get-activity-logs.api.js

```diff
@@ -49,7 +49,7 @@ app.get("/api/clients/:clientId/logs", (req, res, next) => {
     }
 
     logs.forEach((log) => {
-      log.tags = JSON.parse(log.tags)
+      log.tags = log.tags
```

Comment:
In the mysql2 package, `json_arrayagg(json_object())` comes back as a js object rather than a string, like it did in the mysql package